mlflow/store/db_migrations/env.py

```diff
@@ -3,13 +3,16 @@
 from alembic import context
 from sqlalchemy import engine_from_config, pool
 
+from mlflow.environment_variables import MLFLOW_CONFIGURE_LOGGING
+
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
 config = context.config
 
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
-fileConfig(config.config_file_name, disable_existing_loggers=False)
+if MLFLOW_CONFIGURE_LOGGING.get():
+    fileConfig(config.config_file_name, disable_existing_loggers=False)
```

Comment:
Do we actually need `fileConfig`? Can we comment it out? I think it should work.

Reply:
Yes, if we remove the test fails

Reply:
> only controls how loggers are configured.

Right, so it can prevent alembic from overwriting the logging configuration, right?

Reply:
Yes, if we remove this line then logging is not overwritten even MLFLOW_CONFIGURE_LOGGING is True

Reply:
The logging sections, e.g. `[logger_root]`, cannot be removed from the alembic.ini, for alembic requiring them in its Alembic Config object, as far as I know.

When I played around with it before opening the issue, I removed the `fileConfig` line from env.py and instead added respective loggers ("sqlalchemy" and "alembic") to mlflow's own `mlflow.utils.logging_utils._configure_mlflow_loggers`, though that could be omitted. That way, mlflow's logging configuration is not spread across the project and hard to maintain.

Leaving the alembic.ini intact is reasonable for [manual migrations](https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.md#database-schema-changes) and receiving logs for them.

Reply:
@szapp Thanks for the comment.

> When I played around with it before opening the issue, I removed the fileConfig line from env.py and instead added respective loggers ("sqlalchemy" and "alembic") to mlflow's own mlflow.utils.logging_utils._configure_mlflow_loggers, though that could be omitted. That way, mlflow's logging configuration is not spread across the project and hard to maintain.

I think this is a nice middle ground to avoid behavior change and avoid spreading across the project and hard to maintain